notebooks/toy/dialect.py

```diff
@@ -164,6 +169,14 @@ def from_callable(name: str,
             Region.from_block_list([Block.from_callable(input_types, func)]),
             private=private)
 
+    @staticmethod
```

Comment:
Note that this is just a reordering, there are no other changes to the definitions of the operations in this file